core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -366,6 +366,40 @@
       </div>
     </script>
 
+    <script type="text/ng-template" id="modals/saveValidationFail">
+        <div class="modal-header">
+            <h3>Error in saving</h3>
+        </div>
+
+        <div class="modal-body">
+            <p>
+                Sorry, an error has occurred. Please refresh the page to get back to the most recent ‘good’ state.
+            </p>
+        </div>
+
+        <div class="modal-footer">
+            <button class="btn btn-default" ng-click="close()">Close</button>
+        </div>
+    </script>
+
+    <script type="text/ng-template" id="modals/saveVersionMismatch">
+        <div class="modal-header">
+            <h3>Error in saving</h3>
+        </div>
+
+        <div class="modal-body">
+            <p>
+                Sorry, the changes could not be saved. It's possible that someone else has committed to the exploration and so these changes have been lost.
```

Comment:
Updated the text - https://github.com/oppia/oppia/pull/1802/commits/20ef813718d30d2de5d510059be1b6c9c514b624